builder/llpcBuilderImplArith.cpp

```diff
@@ -972,6 +972,16 @@ Value* BuilderImplArith::FDivFast(
                      });
 }
 
+// =====================================================================================================================
+// Create "isInfinite" operation: return true if the supplied FP (or vector) value is infinity
+Value* BuilderImplArith::CreateIsInfinite(
+    Value*        pInValue,   // [in] Input value
+    const Twine&  instName)   // [in] Name to give instruction(s)
+{
+    // 0x004: negative infinity; 0x200: positive infinity
+    return CreateCallAmdgcnClass(pInValue, 0x204, instName);
```

Comment:
please add a bit-field struct or enums to represent the bits of amdgcn.class in llpcIntrinsDefs.h